ansible/configs/sap-hana/default_vars_ec2.yml

```diff
@@ -0,0 +1,332 @@
+
+rootfs_size_bastion: 200
+rootfs_size_tower: 200
+rootfs_size_s4hana: 200
+rootfs_size_hana: 200
+
+security_groups:
+  - name: WebSG
+    rules:
+      - name: HTTPPorts
+        description: "HTTP Public"
+        from_port: 80
+        to_port: 80
+        protocol: tcp
+        cidr: "0.0.0.0/0"
+        rule_type: Ingress
+
+      - name: HTTPSPorts
+        description: "HTTP Public"
+        from_port: 443
+        to_port: 443
+        protocol: tcp
+        cidr: "0.0.0.0/0"
+        rule_type: Ingress
+
+  - name: HostSG
+    rules:
+      - name: FromHostSG
+        description: "Allow everything from HostSG nodes"
+        from_port: 0
+        to_port: 65535
+        protocol: -1
+        from_group: HostSG
+        rule_type: Ingress
+
+instances:
+- name: "bastion"
+  count: 1
+  unique: true
+  public_dns: true
+  dns_loadbalancer: true
+  floating_ip: true
+  image: "{{ bastion_instance_image }}"
+  flavor:
+    ec2: "{{ bastion_instance_type }}"
+  tags:
+    - key: "AnsibleGroup"
+      value: "bastions"
+    - key: "ostype"
+      value: "linux"
+    - key: ansible_python_interpreter
+      value: /usr/libexec/platform-python
+  rootfs_size: "{{ rootfs_size_bastion }}"
+  volumes:
+    - name: /dev/sdf
+      size: 100
+  security_groups:
+    - HostSG
+    - BastionSG
+
+- name: "tower"
+  count: 1
+  unique: true
+  public_dns: true
+  dns_loadbalancer: true
+  floating_ip: true
+  image: "{{ tower_instance_image }}"
+  flavor:
+    ec2: "{{ tower_instance_type }}"
+  tags:
+    - key: "AnsibleGroup"
+      value: "towers"
+    - key: "ostype"
+      value: "linux"
+    - key: ansible_python_interpreter
+      value: /usr/libexec/platform-python
+  rootfs_size: "{{ rootfs_size_tower }}"
+  security_groups:
+    - HostSG
+    - BastionSG
+    - WebSG
+
+- name: "s4hana"
+  count: 1
+  unique: true
+  public_dns: false
+  dns_loadbalancer: false
+  floating_ip: false
+  image: "{{ s4hana_instance_image }}"
+  flavor:
+    ec2: "{{ s4hana_instance_type }}"
+  tags:
+  - key: "AnsibleGroup"
+    value: "s4hanas"
+  - key: "ostype"
+    value: "linux"
+  - key: ansible_python_interpreter
+    value: /usr/libexec/platform-python
+  rootfs_size: "{{ rootfs_size_s4hana }}"
+  volumes:
+    - name: /dev/sdf
+      size: 200
+  security_groups:
+  - HostSG
+
+- name: "hana"
+  count: 2
+  unique: false
+  public_dns: false
+  dns_loadbalancer: false
+  floating_ip: false
+  image: "{{ hana_instance_image }}"
+  flavor:
+    ec2: "{{ hana_instance_type }}"
+  tags:
+  - key: "AnsibleGroup"
+    value: "hanas"
+  - key: "ostype"
+    value: "linux"
+  - key: ansible_python_interpreter
+    value: /usr/libexec/platform-python
+  rootfs_size: "{{ rootfs_size_hana }}"
+  volumes:
+    - name: /dev/sdf
+      size: 500
+  security_groups:
+  - HostSG
+
+common_packages:
+  - unzip
+  - bash-completion
+  - tmux
+  - bind-utils
+  - wget
+  - git
+  - vim-enhanced
+  - at
+
+### Ansible Tower default variables ###
+
+ansible_tower:
```

Comment:
can't you move `ansible_tower` to `default_vars.yml` ?

And have a variable `ansible_tower_disks` that would be different in `default_vars_osp.yml` and `default_vars_ec2.yml` ?